package.json

```diff
@@ -164,6 +168,10 @@
 		"@reach/router": {
 			"react": "$react",
 			"react-dom": "$react-dom"
+		},
+		"@woocommerce/admin-layout": {
+			"react": "^18.2.0",
+			"react-dom": "^18.2.0"
```

Comment:
@joshuatf I had to add an `npm` override for the React version for `@woocommerce/admin-layout`. I didn't see any visible side effects 🤞 

```
npm install @woocommerce/admin-layout --save                   
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: elementor@3.25.0
npm ERR! Found: react@18.3.1
npm ERR! node_modules/react
npm ERR!   dev react@"^18.2.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^17.0.2" from @woocommerce/admin-layout@1.1.0
npm ERR! node_modules/@woocommerce/admin-layout
npm ERR!   @woocommerce/admin-layout@"^1.1.0" from the root project
```